stats.go

```diff
@@ -336,6 +404,7 @@ func (ts *timespan) CompleteWithDuration(value time.Duration) {
 }
 
 type statStore struct {
+	// todo: no idea how memory is managed here, when are the map entries ever deleted?
```

Comment:
The assumption is the that stat names are stable over the lifetime of a process, but this is a known issue for processes that incorrectly use this library and cause a cardinality explosion of stat names. See:

* https://github.com/lyft/gostats/pull/158 (original issue)
* https://github.com/lyft/gostats/pull/160 (my proposed fix)